wasm/wazeroir/interpreter.go

```diff
@@ -1222,7 +1223,7 @@ func (it *interpreter) callNativeFunc(f *interpreterFunction) {
 						um := math.Abs(float64(f - u))
 						dm := math.Abs(float64(f - d))
 						h := u / 2.0
-						if um < dm || float32(math.Floor(float64(h))) == h {
+						if um < dm || (um == dm && float32(math.Floor(float64(h))) == h) {
```

Comment:
found a bug in wazeroir interpreter which hasn't been caught by spectest!

Reply:
this is good, though this line is heavy and needs comment or better variable names.